k8s/Makefile

```diff
@@ -134,14 +134,18 @@ test-k8s-celery-beat-deployments:
 	env NEW_RELIC_APP_NAME="" \
 	j2 celery-beat.yaml.j2 | kubeval --strict --ignore-missing-schemas
 
-k8s-hpa:
+k8s-hpa: k8s-deployment-hpa
+
+k8s-deployment-hpa:
 	j2 hpa.yaml.j2 | ${KC} apply -f -
 
-test-k8s-hpa:
+test-k8s-hpa: test-k8s-deployment-hpa
+
+test-k8s-deployment-hpa:
 	j2 hpa.yaml.j2 | kubeval --strict --ignore-missing-schemas
 
-k8s-delete-hpa:
-	${KC} delete --ignode-not-found deploy ${APP_NAME}
+k8s-delete-deployment-hpa:
```

Comment:
Does this need to go in `PHONY`?

Plus where is/was `k8s-delete-hpa` in `PHONY`?

Reply:
Added in 40c39c198a25295c836be1e74e14a49bbb7b9e6f, there was never a `k8s-delete-hpa` target iirc